docs/user/admin.rst

```diff
@@ -75,12 +75,14 @@ Kolibri users can have different roles with respective access to features:
   * View content and have their progress tracked
   * View *Coach Reports* to track progress of other users and usage stats for individual exercises
   * Create/Edit/Delete other **Admins** and **Learners**
+  * Create/Edit/Delete *Classes* and *Groups* and enroll users in them
```

Comment:
I'm a little confused by the italic/bolding conventions. I would kind of expect all "Kolibri concepts" or words with a technical meaning to have similar styling.